packages/bff/src/services/catalogService.ts

```diff
@@ -176,8 +201,8 @@ export function catalogServiceBuilder(
       return response;
     },
     getProducerEServiceDescriptor: async (
-      eServiceId: string,
-      descriptorId: string,
+      eserviceId: EServiceId,
```

Comment:
```suggestion
      eServiceId: EServiceId,
```
with this renaming we can assign it to the params with the shorthand

Reply:
eserviceId it's our naming convention